clouddriver-ecs/src/main/java/com/netflix/spinnaker/clouddriver/ecs/model/EcsServerGroup.java

```diff
@@ -0,0 +1,210 @@
+/*
+ * Copyright 2017 Lookout, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.clouddriver.ecs.model;
+
+import com.netflix.spinnaker.clouddriver.model.Instance;
+import com.netflix.spinnaker.clouddriver.model.ServerGroup;
+
+import java.util.Map;
+import java.util.Set;
+
+public class EcsServerGroup implements ServerGroup {
+
+  String name;
+  String type;
+  String cloudProvider;
+  String region;
+  Boolean isDisabled;
+  Long createdTime;
+  Set<String> zones;
+  Set<Instance> instances;
+  Set<String> loadBalancers;
+  Set<String> securityGroups;
+  Map<String, Object> launchConfig;
+  InstanceCounts instanceCounts;
+  Capacity capacity;
+  ImagesSummary imagesSummary;
+  ImageSummary imageSummary;
+
+  public EcsServerGroup() {
+
+  }
+
+  @Override
+  public String getName() {
+    return name;
+  }
+
+  @Override
+  public String getType() {
+    return type;
+  }
+
+  @Override
+  public String getCloudProvider() {
+    return cloudProvider;
+  }
+
+  @Override
+  public String getRegion() {
+    return region;
+  }
+
+  public Boolean getDisabled() {
```

Comment:
should this be moved to isDisabled()?

Reply:
Yes.  The convention is that booleans are isXXXXX() instead of getXXXX().  Not sure why it was auto-generated into a getter field though

Reply:
ah, the problem is that the boolean is called `isDisabled`, while it should be called `disabled`, which would respect the java conventions and also the IDE expectations.  Blame this one on netflix :)